src/components/technologies/technology.py

```diff
@@ -731,13 +798,9 @@ def bind_disjunctions(dis):
 
         # CAPEX
         if self.existing and not self.decommission:
-            b_tec.var_capex = Param(domain=Reals, initialize=0)
+            pass
         else:
```

Comment:
get rid of the pass statement: if (self.existing) and (self.decommission):

Reply:
I didn't change this, are we sure we do not need the line that was previously there anymore?

Reply:
We need to have the line again

Reply:
actually we don't, this was the bug ;')